### DISCUSSION TITLE
utiliser le pré-remplissage pour faire une liste déroulante dynamique

### DISCUSSION THREAD

**Discussion by florentarneodo** [MTESSDev/FRW#189] [Annonces]
Bonjour,

Est-ce que vous avez un exemple pour nous aider sur comment utiliser le pré-remplissage ainsi :

Je vais essayer d'être clair :)

À Retraite Québec, un client peut être participant à un ou n régime. Lors de sa demande de prestation de retraite, on peut lui envoyer en pré-remplissage la liste des régimes sur lesquelles on veut qu'il fasse sa demande.

On veut donc pouvoir faire :

Robert crée le formulaire, il est participant à RégimeB et RégimeC.
Il voit une liste déroulante avec seulement RégimeB et RégimeC dans la liste en première question.
Selon ce qu'il choisit il a des questions personnalisées au RégimeB ou RégimeC (ça on sait faire)

Ça marcherait comment pour passer du pré-remplissage à cette liste déroulante? (ou alternative à la liste déroulante dans la même veine).

Merci pour votre aide :)

Florent

**Comment by LarcM7** [comment]
Bonjour, je vous invite à consulter cette page je crois que c'est exactement ce dont vous avez besoin : https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=DomainesValeurs

Complément d'information ici : https://github.com/MTESSDev/FRW/blob/main/Documentation/pre-remplissage.md

Bonne lecture.
